_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss

```diff
@@ -0,0 +1,48 @@
+.component {
+    &.rhd-c-two-col-hero-layout {
+        > .pf-l-grid {
+            
+            > .pf-m-12-col {
+                .component:not(.dark):not(.assembly-slim) {
+                    padding: 0;
+                }
+                > .component {
+                    &.rhd-c-card-grid {
+                        .rhd-c-card-grid__wrapper {
+                            align-items: flex-start;
+                            justify-content: space-between;
+                            margin-right: calc(-1 * var(--pf-l-flex--spacer--sm));
+                            margin-left: calc(-1 * var(--pf-l-flex--spacer--sm));
+                            .pf-c-card {
+                                min-width: 200px;
+                                margin-left: var(--pf-l-flex--spacer--sm);
+                                margin-right: var(--pf-l-flex--spacer--sm);
+                                flex: 1 1 auto;
+                            }
+                        }
+                    }
+                }
+                &:first-of-type {
+                    @media screen and (max-width: $pf-global--breakpoint--lg) {
+                        grid-row: 2;
+                    }
+                    .component {
+                        margin-top: 0;
+                    }
+                }
+
+            }
+
+            .field--name-field-column-2 {
+                > .assembly.component {
+                  // Remove margin-top from first assembly rendered in 2nd
+                  // column within the Layout 2 col, sidebar left assembly type.
+                  // This makes it match the behavior of the left, 1st column.
+                  @media screen and (max-width: $pf-global--breakpoint--lg) {
```

Comment:
I accidentally used the wrong media query logic here. I wanted the opposite and didn't notice at the time of pushing the last commit. I will fix this now.